applications/ecallmgr/src/ecallmgr_fs_bridge.erl

```diff
@@ -158,15 +158,27 @@ handle_bypass_media(DP, _Node, _UUID, #channel{profile=ChannelProfile}, JObj) ->
     end.
 
 -spec maybe_bypass_endpoint_media(kz_json:objects(), ne_binary(), ne_binary(), kz_proplist()) -> kz_proplist().
-maybe_bypass_endpoint_media([Endpoint], BridgeProfile, ChannelProfile, DP) ->
-    EndpointProfile = kz_json:get_value(<<"SIP-Interface">>, Endpoint, BridgeProfile),
-    case kz_json:is_true(<<"Bypass-Media">>, Endpoint)
-        andalso EndpointProfile =:= ChannelProfile of
+maybe_bypass_endpoint_media(Endpoints, BridgeProfile, ChannelProfile, DP) ->
+    BypassOnAll = lists:foldl(fun(EP, 'none') ->
+                                      case bypass_endpoint_media_enabled(EP, BridgeProfile, ChannelProfile) of
+                                          'true' -> 'true';
+                                          'false' -> 'none'
+                                      end;
+                                 (EP, BypassOnAll1) ->
+                                      BypassOnAll1
+                                          andalso bypass_endpoint_media_enabled(EP, BridgeProfile, ChannelProfile)
+                              end, 'none', Endpoints),
+    case BypassOnAll of
         'true' -> [{"application", "set bypass_media=true"}|DP];
-        'false' -> DP
-    end;
-maybe_bypass_endpoint_media(_, _, _, DP) ->
-    DP.
+        'false' -> DP;
+        'none' -> DP
+    end.
+
+-spec bypass_endpoint_media_enabled(kz_json:object(), ne_binary(), ne_binary()) -> boolean().
+bypass_endpoint_media_enabled(Endpoint, BridgeProfile, ChannelProfile) ->
+    EndpointProfile = kz_json:get_value(<<"SIP-Interface">>, Endpoint, BridgeProfile),
```

Comment:
If this is json, `=:=` cannot be used.
If this is an `ne_binary()`, please use `get_ne_binary_value`.